docroot/themes/humsci/humsci_basic/src/scss/tools/_functions.color-pairings.scss

```diff
@@ -1,11 +1,19 @@
-// Map through all color pairings to find and set a color palette
-@function hb-get-pairing-color($key, $palette, $theme-map) {
+@function hb-get-pairing-list($theme-map, $palette) {
```

Comment:
was this function mostly just written for the new hb-pairing-custom-properties mixin? I think you might be able to use the `get-theme-array-list` I wrote in the general.functions.scss in a PR or two ago instead!